packages/amplify-provider-awscloudformation/src/upload-appsync-files.ts

```diff
@@ -1,16 +1,16 @@
-const fs = require('fs');
-const fsext = require('fs-extra');
-const path = require('path');
-
-const TransformPackage = require('graphql-transformer-core');
-const { S3 } = require('./aws-utils/aws-s3');
-const { fileLogger } = require('./utils/aws-logger');
+import fs from 'fs-extra';
+import path from 'path';
+import ora from 'ora';
+import crypto from 'crypto';
+import globby from 'globby';
+import { stateManager, JSONUtilities } from 'amplify-cli-core';
+import { uploadAPIProject } from 'graphql-transformer-core';
+import { S3 } from './aws-utils/aws-s3';
+import { fileLogger } from './utils/aws-logger';
+import { ProviderName as providerName } from './constants';
```

Comment:
Don't alias the import as it adds to confusion when reading code, perhaps it is a sideeffect of refactor?